src/redux/books/books.jsx

```diff
@@ -0,0 +1,95 @@
+/* eslint-disable no-param-reassign */
+import { createAsyncThunk, createSlice } from '@reduxjs/toolkit';
+
+const baseURL = 'https://us-central1-bookstore-api-e63c8.cloudfunctions.net/bookstoreApi/apps/';
+const APID = '6jFLTwMRKqyHQ7J7Rg0F';
+const fullURL = `${baseURL + APID}/books`;
+const BASEACTION = 'bookstore/books/';
+const ADD = `${BASEACTION}/ADD`;
+const REMOVE = `${BASEACTION}/REMOVE`;
+const BOOKFETCH = `${BASEACTION}/GET`;
+
+// Fetchbooks
+const fetchBooks = createAsyncThunk(BOOKFETCH, async () => {
+  const res = await fetch(fullURL);
+  const data = await res.json();
+  // eslint-disable-next-line camelcase
```

Comment:
- [ ] Same here, Please make sure you fix all linter errors instead of disabling them, A linter is great for identifying errors when you use standard rules. Remember, a linter analyzes your code for stylistic and programming errors against the rules it knows. 💯 😉